### PR TITLE
Allow multiple axes in `expand_dims`

### PR DESCRIPTION
**Pull request by toslunar**
Follow numpy/numpy#14051.

Part of #2991.

### CODE REVIEW

File: cupy/core/__init__.py [hunk 1]
```diff
@@ -13,6 +13,7 @@
 from cupy.core._routines_manipulation import broadcast  # NOQA
 from cupy.core._routines_manipulation import broadcast_to  # NOQA
 from cupy.core._routines_manipulation import concatenate_method  # NOQA
+from cupy.core._routines_manipulation import _expand_dims  # NOQA
```
Comment (niboshi) [inline cupy/core/__init__.py]: I think this alias is not necessary.
You can directly `from cupy.core import _routines_manipulation as _manipulation`.

**Comment by niboshi** [comment]
LGTM (just a quick look)

**Comment by toslunar** [comment]
I'll add a test for `axis=()`.

**Comment by toslunar** [comment]
Done.

**Comment by emcastillo** [comment]
Jenkins, test this please

**Pull request merged**